peps/pep-0749.rst

```diff
@@ -74,6 +76,9 @@ We suggest the following deprecation plan:
 
 - In Python 3.14, ``from __future__ import annotations`` will continue to work as it
   did before, converting annotations into strings.
+  - If the future import is active, the ``__annotate__`` function of objects with
```

Comment:
I think this will fix the lint:

```suggestion
   - If the future import is active, the ``__annotate__`` function of objects with
```

Reply:
Thanks, I just pushed a different change that seems to render correctly.